### DISCUSSION TITLE
how to set text_edit_singleline max length,

### DISCUSSION THREAD

**Discussion by skydig** [emilk/egui#1094] [Q&A] [answered]
thanks.

**Comment by emilk** [comment, marked as answer]
Have you tried https://docs.rs/egui/latest/egui/widgets/struct.TextEdit.html#method.desired_width ?

**Reply by skydig** [reply]
desired_width works，thanks，emilk。

**Reply by zhifu3158** [reply]
**@skydig  大佬，请问您那rust编译器报错汉化翻译是怎么做到的？**
vs code 错误提示（语句错误。或变量错误等）如何汉化？或者从什么地方进行汉化修改呢？？可否告知汉化方法？
